imap/jmap_backup.c

```diff
@@ -809,7 +809,7 @@ static int restore_addressbook_cb(const mbentry_t *mbentry, void *rock)
         /* Store the group vCard of recreated contacts */
         r = carddav_store(*mailboxp, crock->group_vcard, NULL, 0, NULL, NULL, 
                           rrock->req->accountid, rrock->req->authstate,
-                          /*ignorequota*/ 0);
+                          /*ignorequota*/ 0, /*oldsize*/ INT_MAX);
```

Comment:
I don't expect Cyrus to run properly on 32-bit systems anyway, but since the old size type is `uint32_t` I suggest to also `UINT32_MAX` here.

Reply:
Fixed